UnityClient/Assets/Scripts/PauseMenu.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PauseMenu : MonoBehaviour
+{
+
+    private bool GamePaused = false; /*Declare GamePaused variable, initially set to false*/
+
+    [SerializeField]
+    private GameObject pauseMenuUI;
+
+    [SerializeField]
+    private void Resume()
```

Comment:
This attribute does nothing for methods. Clue's in the name `SerializeField`. Methods are not fields. Please remove.